antora-ui-camel/src/css/base.css

```diff
@@ -83,3 +83,11 @@ mark {
 .text-center {
   text-align: center;
 }
+
+@media screen (max-width: 1023px) and (min-width: 480px) {
+  code,
+  kbd,
+  pre {
+    overflow-wrap: break-word;
```

Comment:
Does it make sense to remove the media query and put this in the existing rule regardless of the screen size?

Reply:
Yes, it will work.